rotkehlchen/tests/unit/test_globaldb_upgrades.py

```diff
@@ -29,19 +29,48 @@
 ASSETS_IN_V2_GLOBALDB = 3095
 
 
-def _count_v2_v3_assets_inserted() -> int:
-    """Counts and returns how many assets are to be inserted by globaldb_v2_v3_assets.sql"""
-    assets_inserted_by_update = 0
+def _count_sql_file_sentences(file_name: str, skip_tables: int = 0):
```

Comment:
shouln't it be skip_statements? As there is no guarantee two different statements don't touch the same table?

Reply:
Yes. I had one table per statmenent but makes more sense to rename it